doc/classes/ProjectSettings.xml

```diff
@@ -60,6 +60,16 @@
 				Clears the whole configuration (not recommended, may break things).
 			</description>
 		</method>
+		<method name="get_autoload_list" qualifiers="const">
+			<return type="Dictionary[]" />
+			<description>
+				Returns an [Array] of registered autoload classes. Each class is represented as a [Dictionary] that contains the following entries:
```

Comment:
Worth noting that autoloads can either be a Script inheriting **Node**, or a **PackedScene**. Either way, they're always nodes, so "class" may be a bit of a misnomer.